distribution/ddf-common/src/main/resources/common-bin.xml

```diff
@@ -33,6 +33,7 @@
         <exclude>etc/org.apache.karaf.log.cfg</exclude>
         <exclude>etc/org.ops4j.pax.web.cfg</exclude>
         <exclude>etc/users.properties</exclude>
+        <exclude>etc/startup.properties</exclude>
```

Comment:
We used to overwrite files from Karaf but we did include a unique string comment to make it easier to find files that need to be double checked during upgrades.  Eventually we removed all of those files to make upgrading less error prone.  We only did this for files that overwrite karaf distro files and would cause issues like @tbatie mentioned.